flow/migration/v0_to_v1.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Migrate from schema version 0 to version 1.
+
+This migration is a null-migration that serves as a template
+for future migrations and testing purposes.
+"""
+
+
+def migrate_v0_to_v1(project):
+    """Migrate from schema version 0 to version 1."""
+    pass  # nothing to do here, serves purely as an example
+
+
+def migrate_v1_to_v2(project):
+    """Migrate from schema version 1 to version 2."""
+    # TODO: This migration is not yet implemented, but this comment documents
+    # what this function will need to do. There are a few different scenarios
+    # that must be accounted for:
+    # 1. User updates signac before flow. In this case, signac's config API may
+    #    already have changed, and we cannot rely on it. Therefore, we need add
+    #    configobj as an optional requirement for flow rather than relying on
+    #    signac's config API (it's available on PyPI). Alternatively, we could
+    #    bundle configobj like we do in signac, but I don't want to commit to
+    #    that long term so that would at best be a short term solution (say for
+    #    one or two flow releases) and then after that we tell users who
+    #    haven't migrated to just `pip install configobj`. Then, this function
+    #    can rely on the fact that in signac schema version one the config
```

Comment:
Does "version one" mean "v1" here? Just unclear about why it's spelled out.

Reply:
No reason, just stream of consciousness :) I'd appreciate commentary on whether my overall logic here makes sense, though.

Reply:
I think it made sense.